### ISSUE TITLE
Search doesn't work on /guide/web

### ISSUE THREAD

**Issue by LeBenLeBen** [frctl/fractal-docs#16]
1. Go to https://fractal.build/guide/web
2. Try to search
3. Nothing happens

There's an `Invalid object` error in the console coming from Algolia's script.

**Comment by samuelgoddard** [comment]
Well that's pretty weird.... I'm getting the following console error _only_ on that page

```
RangeError: Maximum call stack size exceeded
    at o (57e77f255bc3fc44da368ad5599db84dd458980c.js:1)
    at f (57e77f255bc3fc44da368ad5599db84dd458980c.js:1)
    at r (57e77f255bc3fc44da368ad5599db84dd458980c.js:1)
    at jo (57e77f255bc3fc44da368ad5599db84dd458980c.js:1)
    at e.Mo.confirmTransition (57e77f255bc3fc44da368ad5599db84dd458980c.js:1)
    at e.Mo.transitionTo (57e77f255bc3fc44da368ad5599db84dd458980c.js:1)
    at e.push (57e77f255bc3fc44da368ad5599db84dd458980c.js:1)
    at 57e77f255bc3fc44da368ad5599db84dd458980c.js:1
    at 57e77f255bc3fc44da368ad5599db84dd458980c.js:1
    at f (57e77f255bc3fc44da368ad5599db84dd458980c.js:1)
```

I'll take a look, odd that it's only happening on that single page

**Comment by LeBenLeBen** [comment]
Fixed by #18
